crm_rma_location_rma/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+CRM RMA Location
+================
+
+This module adds a warehouse's stock location for RMA movements in the warehouse and 3 picking types for RMA operations, these picking types are one for Incoming Picking, another for Internal and the third for Outgoing picking. Creating it also document sequences for each of them.
+
+Installation
+============
+
+To install this module, just select it from available modules
+
+Configuration
+=============
+
+No extra configuration is added to this module
+
+Usage
+=====
+
+* Go to Warehouses > Configuration > Warehouses and create a new one
+* Then go to Warehouse > Configuration > Types of Operation, and confirm that it
+  have been created the three mentioned picking types.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/145/8.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* No issues are known yet
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/rma/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/rma/issues/new?body=module:%20crm_rma_location_rma%0Aversion:%208.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Ok, I think then the template should specify this instead. I've made a PR (https://github.com/OCA/rma/pull/57) doing this for modules